doc/whats-new.rst

```diff
@@ -171,7 +171,10 @@ Internal Changes
   ``pytest.raises(Exception, match="foo")``;
   (:pull:`5188`), (:pull:`5191`).
   By `Maximilian Roos <https://github.com/max-sixty>`_.
-
+- Explicit indexes refactor: add an ``xarray.Index`` base class and
+  ``Dataset.xindexes`` / ``DataArray.xindexes`` properties. Also rename
+  ``PandasIndexAdapter`` to ``PandasIndex``, which now inherits from
+  ``xarray.Index`` (:pull:`5102`). By `Benoit Bovy <https://github.com/benbovy>`_.
```

Comment:
This needs to get moved to a new section. But I wouldn't worry about this yet -- it's worth explaining when this functionality is more complete!

Reply:
Yes I plan to add a new section and work on the documentation as soon as users can start leveraging Xarray's explicit/flexible indexes through public API (well, `xindexes` is new API, but it doesn't bring much value yet).

Reply:
in any case, could you move this to the appropriate section of the next release?